tests/framework/installer/ceph_manifests_v1.5.go

```diff
@@ -336,7 +336,34 @@ spec:
     active: ` + strconv.Itoa(count)
 }
 
-func (m *CephManifestsV1_5) GetObjectStore(name string, replicaCount, port int) string {
+func (m *CephManifestsV1_5) GetObjectStore(name string, replicaCount, port int, tlsEnable bool) string {
+	if tlsEnable {
```

Comment:
Looks like the upgrade test only calls `runObjectE2ETestLite()`, which does not use tls. In this case, no need to implement this for the 1.5 manifests.